### ISSUE TITLE
Triplet loss training - how long does it take?

### ISSUE THREAD

**Issue by LillyBlackWhite** [davidsandberg/facenet#647]
Hello, @davidsandberg 

I was wondering how long did it take you to train your neural network in order to achieve these great results?
Did you train the network from scratch? (reading the code, I think you trained it from scatch, without using a pre-trained model, but I just wanted to be sure)
